src/coreclr/vm/interpexec.cpp

```diff
@@ -3618,6 +3733,307 @@ do                                                                      \
                 case INTOP_THROW_PNSE:
                     COMPlusThrow(kPlatformNotSupportedException);
                     break;
+
+                case INTOP_HANDLE_CONTINUATION_GENERIC:
+                case INTOP_HANDLE_CONTINUATION:
+                {
+                    int32_t helperOffset = *ip == INTOP_HANDLE_CONTINUATION_GENERIC ? 4 : 3;
+                    int32_t ipAdjust = *ip == INTOP_HANDLE_CONTINUATION_GENERIC ? 5 : 4;
+                    int32_t suspendDataOffset = *ip == INTOP_HANDLE_CONTINUATION_GENERIC ? 3 : 2;
+                    InterpAsyncSuspendData *pAsyncSuspendData = (InterpAsyncSuspendData*)pMethod->pDataItems[ip[suspendDataOffset]];
+
+                    // Zero out locals that need zeroing
+                    InterpIntervalMapEntry *pZeroLocalsEntry = pAsyncSuspendData->zeroedLocalsIntervals;
+                    while (pZeroLocalsEntry->countBytes != 0)
+                    {
+                        memset(LOCAL_VAR_ADDR(pZeroLocalsEntry->startOffset, uint8_t), 0, pZeroLocalsEntry->countBytes);
+                        pZeroLocalsEntry++;
+                    }
+
+                    if (pInterpreterFrame->GetContinuation() == NULL)
+                    {
+                        // No continuation to handle.
+                        LOCAL_VAR(ip[1], void*) = NULL; // We don't allocate a continuation here
+                        ip += ipAdjust; // (4 for this opcode)
```

Comment:
So, we never get here for the INTOP_HANDLE_CONTINUATION_GENERIC? Asking because of the comment

Reply:
Sigh, yes we get there... comment updated to indicate we might move 4 or 5.